command/v6/service_access_command_test.go

```diff
@@ -138,9 +140,98 @@ var _ = Describe("service-access Command", func() {
 					nil)
 			})
 
-			It("displays flavor text", func() {
-				Expect(testUI.Out).To(Say("Getting service access as some-user..."))
-				Expect(fakeConfig.CurrentUserCallCount()).To(Equal(1))
+			Describe("flavour text", func() {
```

Comment:
I'd suggest [`DescribeTable`](https://godoc.org/github.com/onsi/ginkgo/extensions/table#DescribeTable) as an option for making these tests a bit cleaner.

Reply:
Yeah thats a good point, we will reduce these Describe down to a DescribeTable and see how it looks.

Reply:
Done.